paz/processors/standard.py

```diff
@@ -482,3 +482,14 @@ def __init__(self, topics):
     def call(self, dictionary):
         [print(dictionary[topic]) for topic in self.topics]
         return dictionary
+
+
+class ComputeSelectedIndices(Processor):
+    """Computes row-wise intersection between two given
+    arrays and returns the indices of the intersections.
+    """
+    def __init__(self):
+        super(ComputeSelectedIndices, self).__init__()
+
+    def call(self, box_data_raw, box_data):
+        return compute_selected_indices(box_data_raw, box_data)
```

Comment:
We have to look if this is necessary? If so maybe a different name would also be better.